### ISSUE TITLE
stderr maxBuffer exceeded

### ISSUE THREAD

**Issue by lmeyerov** [lerna/lerna#466]
When running bootstrap via lerna, I get the following error:
```
leoretina:viz-app lmeyerov$ npm run build

> @ build /Users/lmeyerov/Desktop/work/viz-app
> lerna bootstrap

Lerna v2.0.0-beta.31
Bootstrapping 2 packages
Preinstalling packages
Installing external dependencies
Symlinking packages and binaries
Postinstalling packages
Prepublishing packages
NpmUtilities.execInDir        ("run prepublish", [], "./packages/viz-app")
Error: stderr maxBuffer exceeded.  Partial output follows:

 69% building modules 1433/1442 modules 9 active ...iz-client/components/labels/labels.js  
NpmUtilities.runScriptInDir   ("prepublish", [], "./packages/viz-app")
Error: stderr maxBuffer exceeded.  Partial output follows:

 69% building modules 1433/1442 modules 9 active ...iz-client/components/labels/labels.js  
Errored while running BootstrapCommand.execute
Error: stderr maxBuffer exceeded.  Partial output follows:

 69% building modules 1433/1442 modules 9 active ...iz-client/components/labels/labels.js  

npm ERR! Darwin 13.3.0
npm ERR! argv "/Users/lmeyerov/.nvm/versions/node/v6.6.0/bin/node" "/Users/lmeyerov/.nvm/versions/node/v6.6.0/bin/npm" "run" "build"
npm ERR! node v6.6.0
npm ERR! npm  v3.10.3
npm ERR! code ELIFECYCLE
npm ERR! @ build: `lerna bootstrap`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @ build script 'lerna bootstrap'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the  package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     lerna bootstrap
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs 
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls 
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/lmeyerov/Desktop/work/viz-app/npm-debug.log
```

This is surprising because the build succeeds for that package when I run it directly via `npm run build`:

```
leoretina:viz-app lmeyerov$ npm run build

> @graphistry/viz-app@2.7.6 build /Users/lmeyerov/Desktop/work/viz-app/packages/viz-app
> npm run clean && cross-env NODE_ENV=production node ./tool/build


> @graphistry/viz-app@2.7.6 clean /Users/lmeyerov/Desktop/work/viz-app/packages/viz-app
> rm -rf ./www/*

[WEBPACK] 🔨  Started building viz-client.js
[WEBPACK] 🔨  Started building viz-server.js                                                
[WEBPACK] ✅  Successfully built viz-client.js                                           
[WEBPACK] ✅  Successfully built viz-server.js
```

Unfortunately I cannot share the sources, but am happy to (try to) provide further information as requested.

**Comment by lmeyerov** [comment]
Ah, looking at `lerna-debug.log`:

```
...
\n[WEBPACK] ✅  Successfully built graphistryJS.js\n"
lerna(error)   NpmUtilities.execInDir        ("run prepublish", [], "./packages/viz-app")
    Error: stderr maxBuffer exceeded.  Partial output follows:
    
      0% compiling 10% building modules 0/1 modules 1 active multi vendor              10% building modules 1/1 modules 0 active 10% building modules 1/2 modules 1 active ...node_modules/moment/min/moment.min.js 10% building modules 1/3 modules 2 active ...phistry/falcor/dist/falcor.all.min.js 10% building modules 1/4 modules 3 active ...viz-app/node_modules/lodash/lodash.js 10% building modules 1/5 modules 4 active ...pp/node_modules/simpleflakes/index.js 10% building modules 1/6 modules 5 
...
picker/styles.less 69% building modules 1432/1436 modules 4 active ...mponents/histograms/EncodingPicker.js 69% building modules 1432/1437 modules 5 active ...nts/histograms/sparklineComponents.js 69% building modules 1433/1437 modules 4 active ...nts/histograms/sparklineComponents.js 69% building modules 1433/1438 modules 5 active ...omponents/tethered-select/styles.less 69% building modules 1433/1439 modules 6 active ...de_modules/react-select/lib/Select.js 69% building modules 1433/1440 modules 7 active ...ed/components/color-pill/colorPill.js 69% building modules 1433/1441 modules 8 active ...viz-client/components/labels/label.js 69% building modules 1433/1442 modules 9 active ...iz-client/components/labels/labels.js
lerna(error)   NpmUtilities.runScriptInDir   ("prepublish", [], "./packages/viz-app")
lerna(error)   Errored while running BootstrapCommand.execute
```

This seems related to other maxBuffer issues posted here, apparently not resolved.

**Comment by danibram** [comment]
same here, when i try to run a webpack task from lerna "2.0.0-beta.37"

**Comment by evocateur** [comment]
Our current `maxBuffer` is 500KiB (`500 * 1024` bytes). Perhaps we should bump that up for lifecycle scripts during `lerna publish`?

Does anyone know what the drawbacks are of specifying an absurdly large number for this config value, aside from potentially egregious memory consumption?

**Comment by noherczeg** [comment]
@evocateur I believe this has been fixed in #782
